components/shell/fish/Makefile

```diff
@@ -0,0 +1,50 @@
+#
```

Comment:
Please, use new template for header (look at https://github.com/OpenIndiana/oi-userland/blob/oi/hipster/templates/configure.mk)